lib/Doctrine/DBAL/Schema/OracleSchemaManager.php

```diff
@@ -288,7 +278,7 @@ protected function _getPortableDatabaseDefinition($database)
     /**
      * {@inheritdoc}
      */
-    public function createDatabase($database = null)
+    public function createDatabase(?string $database = null) : void
```

Comment:
`$database` should be non-nullable same as in the parent method.

Reply:
Do you know why it was allowed to be null for Oracle? I assumed there was a reason and I thought I remembered reading something somewhere in the code about it.

Reply:
Fixed in https://github.com/doctrine/dbal/pull/3565/commits/733e1b6f792adcaf3894cba2a55c25b35686e14b